build/images/Dockerfile.build.windows

```diff
@@ -52,5 +52,7 @@ COPY --from=windows-ovs /Windows/System32/vcruntime140.dll /Windows/System32/
 COPY --from=windows-ovs /Windows/System32/libeay32.dll /Windows/System32/
 COPY --from=windows-ovs /Windows/System32/ssleay32.dll /Windows/System32/
 
+COPY build/images/scripts/* /
```

Comment:
COPY build/images/scripts/* /usr/local/bin/, does the folder  build/images/scripts/* and file exist on windows host machine where this docker file is getting executed , it is better to copy script from image, also copy specific image, we do not needs all scripts

Reply:
build/images/scripts/*  exists in antrea repo , we can copy the specific script .This is usually the preferred way of copying the scripts i.e we normally do it from antrea repo
[antrea/build/images/Dockerfile.build.ubuntu](https://github.com/antrea-io/antrea/blob/9d9fa73afc8383b929d9843773bd2b93b794c323/build/images/Dockerfile.build.ubuntu#L41)